warnings.cpp

```diff
@@ -3,22 +3,29 @@
 Warnings::Warnings(){ return; }
 
 void Warnings::setup(Ui::MainWindow *ui) {
-    this->ui = ui;
-    WarningLabel* battery = new WarningLabel(OFF,QPixmap(":/icons/battery.png"),ui->batteryWarning);
-    WarningLabel* motor= new WarningLabel(OFF,QPixmap(":/icons/motor.png"),ui->motorWarning);
+  this->ui = ui;
+  WarningLabel* battery = new WarningLabel(OFF,QPixmap(":/icons/battery.png"),ui->batteryWarning);
+  WarningLabel* motor= new WarningLabel(OFF,QPixmap(":/icons/motor.png"),ui->motorWarning);
 
-    warning_labels.insert({BATTERY, battery});
-    warning_labels.insert({MOTOR, motor});
-    off(BATTERY);
-    off(MOTOR);
+  warning_labels.insert({BATTERY, battery});
+  warning_labels.insert({MOTOR, motor});
+  
+  this->ui = ui;
+  WarningLabel* battery = new WarningLabel(OFF,QPixmap(":/icons/battery.png"),ui->batteryWarning);
```

Comment:
You can remove the double definition of `BATTERY` and `MOTOR`, there is more than 1 line, so just make sure any code duplication isnt there. You can compile/run it to make sure there isnt any